pkgs/stdenv/generic/setup.sh

```diff
@@ -125,11 +126,6 @@ _eval() {
 ######################################################################
 # Logging.
 
-# Obsolete.
-stopNest() { true; }
-header() { echo "$1"; }
-closeNest() { true; }
```

Comment:
A warning would have been nice before dropping those.

Reply:
I'm always surprised by this kind of out-of-tree usage and the expectation, that anyone in nixpkgs would know.

Reply:
Even without knowing, you can't assume that something is *not* used, if it's been exposed for some time (sadly). You never know how things bubble their way up from the entrails of nixpkgs to cargo-culted code snippets.

Reply:
For my curiosity: how would you notice the warning?

Reply:
Hopefully by looking at build logs. The longer the warnings stay around, the more likely that is to happen.

Reply:
I have high doubts that those warnings inside of the build logs would have been noticed.